main.py

```diff
@@ -192,6 +205,12 @@ def main():
     # Bot error handler
     dispatcher.add_error_handler(error_handler)
 
+    for k, v in config.items():
+        if v == "":
+            logging.warning(
+                f"{k} environment variable not provided, the commands that use this will be disabled."
+            )
+
```

Comment:
Specify exactly which commands are disabled. E.g.:
```bash
[✓] TELEGRAM_BOT_TOKEN: Token found, establishing connection to Telegram...
[×] WOLFRAM_APP_ID: /calc is disabled
[×] CLIMACELL_API_KEY: /weather is disabled
```

On top of that, it would be great if we could get green terminal output for passing cases and red for failing.